script/lobsters-cron

```diff
@@ -19,8 +19,6 @@ fi
 
 bundle exec ruby script/mail_new_activity.rb &
 bundle exec ruby script/mastodon_post.rb &
-bundle exec ruby script/mastodon_sync_users.rb &
```

Comment:
Was this job already broken? I don't see any file at `script/mastodon_sync_users.rb`, but I do see `script/mastodon_sync_list.rb`